biscuit-auth/src/capi.rs

```diff
@@ -310,9 +317,13 @@ pub unsafe extern "C" fn key_pair_new<'a>(
     seed.copy_from_slice(slice);
 
     let mut rng: StdRng = SeedableRng::from_seed(seed);
+    let algorithm = match algorithm {
```

Comment:
a from instance would be nice

Reply:
fixing that one later, because capi had a lot of changes in main, that we first need to forward port to v5